scripts/chocolatey/coder.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Read this before creating packages: https://docs.chocolatey.org/en-us/create/create-packages -->
+<!-- It is especially important to read the above link to understand additional requirements when publishing packages to the community feed aka dot org (https://community.chocolatey.org/packages). -->
+
+<!-- Test your packages in a test environment: https://github.com/chocolatey/chocolatey-test-environment -->
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <!-- == PACKAGE SPECIFIC SECTION == -->
+    <!-- This section is about this package, although id and version have ties back to the software -->
+    <!-- id is lowercase and if you want a good separator for words, use '-', not '.'. Dots are only acceptable as suffixes for certain types of packages, e.g. .install, .portable, .extension, .template -->
+    <!-- If the software is cross-platform, attempt to use the same id as the debian/rpm package(s) if possible. -->
+    <id>coder</id>
+    <version>$version$</version>
+    <packageSourceUrl>https://github.com/coder/coder/blob/main/scripts/chocolatey</packageSourceUrl>
+    <!-- owners is a poor name for maintainers of the package. It sticks around by this name for compatibility reasons. It basically means you. -->
+    <owners>Coder Technologies\, Inc.</owners>
+    <!-- ============================== -->
+
+    <!-- == SOFTWARE SPECIFIC SECTION == -->
+    <!-- This section is about the software itself -->
+    <title>Coder (Install)</title>
+    <authors>Coder Technologies\, Inc.</authors>
+    <projectUrl>https://coder.com</projectUrl>
+    <iconUrl>https://github.com/coder/presskit/raw/main/logos/coder%20logo%20black%20square.png?raw=true</iconUrl>
+    <copyright>Coder Technologies, Inc.</copyright>
+    <licenseUrl>https://github.com/coder/coder/blob/main/LICENSE.enterprise</licenseUrl>
```

Comment:
```suggestion
    <licenseUrl>https://coder.com/legal/terms-of-service</licenseUrl>
```

Reply:
For license we can use this link. I have seen similar approach by [tailscale chocolaty package](https://github.com/tailscale/tailscale-chocolatey/blob/main/tailscale.nuspec#L16). 
cc: @bpmct

Reply:
That works!